schema/release-schema.json

```
@@ -178,6 +178,30 @@
           "items": {
             "$ref": "#/definitions/Milestone"
           }
+        },
+        "finalStatus": {
+          "title": "Planning final status",
+          "description": "The final status of the planning process, from the closed [planningFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#planning-final-status) codelist.",
```

```suggestion
          "description": "The final status of the planning process, using the closed [planningFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#planning-final-status) codelist.",
```

as per the style guide